h2o-persist-gcs/src/main/java/water/persist/PersistGcs.java

```diff
@@ -29,9 +29,22 @@
 /**
  * Persistence backend for GCS
  */
+@SuppressWarnings("unused")
 public final class PersistGcs extends Persist {
 
-  private final Storage storage = StorageOptions.getDefaultInstance().getService();
+  // Value in DefaultCredentialsProvider.NO_GCE_CHECK_ENV_VAR is not publicly accessible
+  private static final String NO_GCE_CHECK_ENV_VAR = "NO_GCE_CHECK";
+
+  private final Storage storage;
+
+  @SuppressWarnings("unused")
```

Comment:
The class is instantiated by means of reflection, these warnings can be supressed.